webpack/base.config.js

```diff
@@ -30,9 +34,9 @@ module.exports = {
   ],
   module: {
     rules: [
-      // we pass the output from babel loader to react-hot loader
```

Comment:
Is this no longer true?

Reply:
We do, I think I removed some comments as I edited the file. I can add back